### PR TITLE
tye run Steeltoe MusicStore

### PR DESCRIPTION
**Pull request by TimHess**
`tye run --docker` works now, its a draft because I'm planning to come back and update this for `netcoreapp3.1`/Steeltoe 3.0

### CODE REVIEW

File: steeltoe-oss/MusicStore/src/MusicStoreService/Program.cs [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
+                .ConfigureAppConfiguration((builderContext, configBuilder) => configBuilder.AddConfigServer(builderContext.HostingEnvironment.EnvironmentName, GetLoggerFactory()))
+                .ConfigureLogging((context, builder) => builder.AddDynamicConsole())
                 .Build();
 
             SeedDatabase(host);
 
             host.Run();
         }
 
+        public static ILoggerFactory GetLoggerFactory()
```
Comment (davidfowl) [inline steeltoe-oss/MusicStore/src/MusicStoreService/Program.cs]: What's this for?

File: steeltoe-oss/MusicStore/src/MusicStoreUI/MusicStoreUI.csproj [hunk 2]
```diff
@@ -27,27 +26,19 @@
     <PackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="$(AspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.14" />
```
Comment (davidfowl) [inline steeltoe-oss/MusicStore/src/MusicStoreUI/MusicStoreUI.csproj]: Why not 3.1?

File: steeltoe-oss/MusicStore/src/MusicStoreUI/Program.cs [hunk 3]
```diff
@@ -16,34 +17,32 @@ public static void Main(string[] args)
...
-                })
+                .ConfigureLogging((context, builder) => builder.AddDynamicConsole())
                 .Build();
 
             SeedDatabase(host, config);
 
             host.Run();
         }
 
+        public static ILoggerFactory GetLoggerFactory()
```
Comment (davidfowl) [inline steeltoe-oss/MusicStore/src/MusicStoreUI/Program.cs]: Same question.

File: steeltoe-oss/MusicStore/src/MusicStoreService/Program.cs (hunk 1 above)
Comment (TimHess) [reply steeltoe-oss/MusicStore/src/MusicStoreService/Program.cs]: Logging inside the config server config provider... I'm definitely open to alternatives

File: steeltoe-oss/MusicStore/src/MusicStoreUI/MusicStoreUI.csproj (hunk 2 above)
Comment (TimHess) [reply steeltoe-oss/MusicStore/src/MusicStoreUI/MusicStoreUI.csproj]: part of the next step... there's a bunch of out of date stuff in the current state

**Pull request closed without merging**